### PR TITLE
fix: #6 passing control limits on entity creation

### PR DESCRIPTION
**Pull request by RadicalAcronym**

### CODE REVIEW

File: safe_autonomy_simulation/entities/integrator.py [hunk 1]
```diff
@@ -50,6 +50,8 @@ def __init__(
         damping=DAMPING_DEFAULT,
         trajectory_samples=0,
         integration_method="RK45",
+        control_min: typing.Union[float, np.ndarray, None] = -1,
+        control_max: typing.Union[float, np.ndarray, None] = 1,
```
Comment (jamie-cunningham) [inline safe_autonomy_simulation/entities/integrator.py:54]: Looks good, but please add lines to the docstring describing the new parameters

File: safe_autonomy_simulation/entities/integrator.py [hunk 2]
```diff
@@ -149,6 +151,8 @@ def __init__(
...
```
Comment (jamie-cunningham) [inline safe_autonomy_simulation/entities/integrator.py:155]: Once again looks good to me but please add to docstring

File: safe_autonomy_simulation/entities/integrator.py [hunk 3]
```diff
@@ -254,6 +258,8 @@ def __init__(
...
```
Comment (jamie-cunningham) [inline safe_autonomy_simulation/entities/integrator.py:262]: Fine defaults and implementation but needs docstring

File: safe_autonomy_simulation/sims/spacecraft/point_model.py [hunk 4]
```diff
@@ -79,6 +79,8 @@ def __init__(
         material: mat.Material = defaults.CWH_MATERIAL,
         parent: typing.Union[e.PhysicalEntity, None] = None,
         children: list[e.PhysicalEntity] = [],
+        control_min: typing.Union[float, np.ndarray, None] = -1,
```
Comment (jamie-cunningham) [inline safe_autonomy_simulation/sims/spacecraft/point_model.py:82]: Looks good but needs docstring

File: safe_autonomy_simulation/sims/spacecraft/rotational_model.py [hunk 5]
```diff
@@ -116,6 +116,8 @@ def __init__(
         material: mat.Material = defaults.CWH_MATERIAL,
         parent: typing.Union[e.PhysicalEntity, None] = None,
         children: list[e.PhysicalEntity] = [],
+        control_min: typing.Union[float, np.ndarray, None] = None,
+        control_max: typing.Union[float, np.ndarray, None] = None,
```
Comment (jamie-cunningham) [inline safe_autonomy_simulation/sims/spacecraft/rotational_model.py:120]: Fine as is being a direct override of anything computed using the angular limits. However this behavior should be noted in a docstring entry for each of these new parameters. Something indicating that if this value is None, defaults computed using angular limits will be used.

**Review by jamie-cunningham — CHANGES_REQUESTED** [review]

File: safe_autonomy_simulation/sims/spacecraft/rotational_model.py (hunk 5 above)
Comment (RadicalAcronym) [reply safe_autonomy_simulation/sims/spacecraft/rotational_model.py:126]: Please check that doc strings have been updated satisfactorily.

**Review by jamie-cunningham — APPROVED** [review]

**Pull request merged**